pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2047,20 +1977,50 @@ def predict_model(
 
         """
 
-        return super().predict_model(
-            estimator=estimator,
-            data=data,
-            probability_threshold=None,
-            encoded_labels=True,
-            round=round,
-            verbose=verbose,
-        )
+        # return super().predict_model(
+        #     estimator=estimator,
+        #     data=data,
+        #     probability_threshold=None,
+        #     encoded_labels=True,
+        #     round=round,
+        #     verbose=verbose,
+        # )
+
+        if fh is None:
+            fh = self.fh
+
+        try:
+            return_vals = estimator.predict(
+                X=data, fh=fh, return_pred_int=return_pred_int, alpha=alpha
+            )
+        except NotImplementedError:
+            # Most likely prediction interval has not been implemented.
+            # Hence try by explicitly setting it to False
+            return_vals = estimator.predict(
+                X=data, fh=fh, return_pred_int=False, alpha=alpha
+            )
```

Comment:
We should log this as a warning

Reply:
I have added warning messages now. Where is this log file captured though?